renderer/components/exports/export.js

```diff
@@ -52,16 +53,25 @@ export default class Export extends React.Component {
     const {menu} = this.state;
 
     const cancelable = status === 'waiting' || status === 'processing';
-
+    const fileNameClassName = classNames({
+      title: true,
+      'title-color': !cancelable,
+      'title-color-disabled': cancelable
+    });
     return (
       <div draggable className="export-container" onClick={this.openFile} onDragStart={this.onDragStart}>
         <div className="thumbnail">
           <div className="overlay"/>
           <div className="icon" onClick={stopPropagation}>
             {
               cancelable ?
-                <CancelIcon fill="white" hoverFill="white" activeFill="white" onClick={cancel}/> :
-                <IconMenu icon={MoreIcon} fill="white" hoverFill="white" activeFill="white" onOpen={menu && menu.popup}/>
+                <div className="icon" onClick={cancel}>
+                  <CancelIcon className="icon" fill="white"
```

Comment:
```suggestion
                  <CancelIcon fill="white"
```

Does this need that class? Since it's on the parent

Reply:
Not needed. I'll remove it.